huego.go

```diff
@@ -85,7 +85,7 @@ func unmarshal(data []byte, v interface{}) error {
 	return nil
 }
 
-func get(ctx context.Context, url string) ([]byte, error) {
+func (b *Bridge) get(ctx context.Context, url string) ([]byte, error) {
```

Comment:
Add a `http.Client` function parameter to the `get`, `post`, `put`... methods rather  than moving those functions to the `Bridge` type as pointer receivers.

Reply:
Why? At the very least I really don't want to put delete back to just being func delete as you're overlapping with a built-in function.

Reply:
Good point. Then we’ll rename it to del so it doesn’t overlap.